app/components/ui/atoms/CrossIconButton.js

```diff
@@ -1,19 +1,9 @@
 import React from 'react'
-import styled from 'styled-components'
-
 import Icon from './Icon'
+import ButtonAsIconWrapper from './ButtonAsIconWrapper'
 
 const CrossIcon = props => <Icon iconName="X" overrideName="cross" {...props} />
 
-const ButtonAsIconWrapper = styled.button.attrs({
-  type: 'button',
-})`
-  background-color: transparent;
-  border: none;
-  line-height: 0;
-  padding: 0;
-`
-
 const CrossIconButton = props => (
```

Comment:
When I created this component, I was debating whether this was worth having (vs just having `ButtonAsIconWrapper`.

Do you think it's still worth having as its own component? I'm tempted to say refactor by removing this component altogether & explicitly wrapping `CrossIcon` in `ButtonAsIconWrapper` in any context where it's necessary.

What do you think @Cooryd ?